packs/data/pfs-season-1-bestiary.db/elite-skeletal-giant-pfs-1-18.json

```diff
@@ -716,7 +716,7 @@
             "type": "lore"
         }
     ],
-    "name": "Elite Skeletal Giant (1-18)",
+    "name": "Elite Skeletal Giant (PFS 1-18)",
```

Comment:
Per Michael Sayre, the PDF is missing the elite adjustment:
https://paizo.com/threads/rzs430w8?PFS2E-118-Lodge-of-the-Living-God#5
> the elite skeletal giants from the higher subtier for encounter C3 are missing their elite adjustments, so please apply those as described in the elite entry until we get an update pushed.

I don't think this update will get pushed, as it hasn't happened yet.

As this is identical to adding the Elite adjustment using the button on the NPC sheet for the Bestiary version of the Skeletal Giant, we can probably get rid of this variant.